src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -45,6 +45,9 @@ public class FunctionInstanceSnapshot
 
         public DateTimeOffset? EndTime { get; set; }
 
+        // For fast-style logging. 
```

Comment:
Can you add a better description for this? What exactly does this value represent